### DISCUSSION TITLE
ZFSBootMenu is stuck at black screen with AM5 AGESA 1.1.8.0

### DISCUSSION THREAD

**Discussion by jxdking** [zbm-dev/zfsbootmenu#645] [Questions] [answered]
ZFSBootMenu is stuck at black screen when booting with AM5 AGESA 1.1.8.0. ZFSBootMenu is listed in UEFI boot list. When selected, it is stuck at black screen forever. 
The same setup does not have issue with old bios before AGESA 1.1.8.0. Tried both UEFI and CSM mode. They exhibit the same issue.

**Comment by darbsh** [comment]
@jxdking Did you manage to find a solution? I just updated to a bios with AGESA 1.2.0.0 in anticipation of the new 9000 series, and now have the same issue.

**Reply by darbsh** [reply, marked as answer]
Figured I'd update this for anyone looking in the future. I booted the separate components from https://get.zfsbootmenu.org/ in Refind, instead of the bundled EFI file, and it's working again.

**Reply by jxdking** [reply]
I ditched zfsbootmenu. In the end, I moved the /boot folder to an ext4 partition and let grub to handle that. Problem solved.
